cloud/ec2_fleet.go

```diff
@@ -373,6 +380,44 @@ func (m *ec2FleetManager) OnUp(ctx context.Context, h *host.Host) error {
 	return nil
 }
 
+func (m *ec2FleetManager) Cleanup(ctx context.Context) error {
+	if err := m.client.Create(m.credentials, m.region); err != nil {
+		return errors.Wrap(err, "error creating client")
+	}
+	defer m.client.Close()
+
+	launchTemplates, err := m.client.GetLaunchTemplates(ctx, &ec2.DescribeLaunchTemplatesInput{
+		Filters: []*ec2.Filter{
+			{Name: aws.String("tag-key"), Values: []*string{aws.String(evergreen.TagDistro)}},
```

Comment:
I went the service flag route.

Reply:
This comment made me think about passing an option to service-web from the command line to use the env files to differentiate between staging and prod. This would have allowed me to tag staging and prod resources differently. I opted not to because I didn't want to add another way staging and prod are different.